helm_chart/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ .Values.label }}-configmap
```

Comment:
Names of resources should be taken from Release name as best practice.
These can be overriden via values but not as labels as they are something else, check how helm create makes files.